<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>763</width>
      <height>460</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="tabHire">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Найм</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_miner">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>110</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рынки:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fermersHouse">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Фермеры:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_marketeer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_horseman">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fermers">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_horseman">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_stables">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>270</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_mine">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>220</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_market">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>140</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_farmers">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_farmersHouse">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>350</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_mine">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>190</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Торговцы:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Хозяева:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Шахты:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stables">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 160px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_miner">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>270</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Конюшни:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_marketeer">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>350</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дома:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Шахтёры:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_market">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>110</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_fermers_MAX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 60px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_horseman_MAX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 60px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_marketeer_MAX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 60px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_miner_MAX">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> border:2px solid #787878;
 color: #000000;

 font-family: Times New Roman;
 width: 60px;
 height: 40px;
 font-size: 20px;
 background-image: linear-gradient(to top, #E3E3E3, #E0E0E0);
 background-color: #B5B5B5;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabMapTown">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Карта Города</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>752</width>
        <height>422</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_mapTown">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>467</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Деньги:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cash">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>467</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addMarketeer">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_change_ectt">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>change ectt</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
